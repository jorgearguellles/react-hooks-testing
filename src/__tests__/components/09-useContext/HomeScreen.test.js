import React from 'react';
import '@testing-library/jest-dom';
import { mount } from 'enzyme';
import { HomeScreen } from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Testing <HomeScreen />', ()=>{

  const user = {
    name: 'Jorge',
    email: 'jorge@gmail.com'
  }
  const wrapper = mount( 
    <UserContext.Provider value={{ user }}>
      <HomeScreen /> 
    </UserContext.Provider>
  );

  test('1. SnapShot', ()=>{
    expect( wrapper ).toMatchSnapshot();
  })
})