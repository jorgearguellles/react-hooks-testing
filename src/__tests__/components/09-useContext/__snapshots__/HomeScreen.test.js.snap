// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <HomeScreen /> 1. SnapShot 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Jorge",
   "email": "jorge@gmail.com"
}
  </pre>
</div>
`;
